adserver/mixins.py

```diff
@@ -255,7 +264,7 @@ def get_context_data(self, **kwargs):  # pylint: disable=missing-docstring
 
         # Sort by date when filtering a specific value,
         # otherwise handle sorting via the users input
-        if filtered:
+        if len(filtered) >= self.FILTER_COUNT:
```

Comment:
I don't really understand this. What is this threshold for? I understand we almost always want to order by date but I don't understand this.

Reply:
Basically there is a report with 2 indexes, so it makes stuff weird :/